flow/project.py

```diff
@@ -785,17 +791,26 @@ def op2(job):
         the directives of the singleton group containing that operation are
         used. To prevent this, set the directives to an empty dictionary for
         that operation.
-    options : str
-        A string of options to append to the output of the object's call method.
-        This allows options like ``--num_passes`` to be given to a group.
-
+    run_options : str
+        The options to pass to ``entrypoint exec`` when running the group. These will only be used
+        when the operation is forked instead of running in the same Python interpretor.
+    group_aggregator : :class:`~.aggregator`
```

Comment:
I think you copied the wrong docstrings from above? `submit_options` is missing and `group_aggregator` shouldn't be listed here.

Reply:
Yep, I did. I am currently fighting off COVID, so I am not at peak mental form. :laughing: